### ISSUE TITLE
[BUG]: Can't download from ADN since the laste update.

### ISSUE THREAD

**Issue by nigoki2** [anidl/multi-downloader-nx#750]
### Program version

5.1.4

### Operating System

Windows

### Type

GUI

### Service

AnimationDigitalNetwork

### Command used

Download

### Show ID

1151

### Episode

1

### Console Output

```Shell
GUI server started on port 3000
[WS] Connection to public ws from '::ffff:127.0.0.1'
[WS] Connection from '::ffff:127.0.0.1'
Authentication Success
[DEBUG] Got resolve options: {"id":"1151","q":2,"e":"1","dubLang":["jpn"],"dlsubs":["fr"],"fileName":"[${service}] ${showTitle} - S${season}E${episode} [${height}p]","dlVideoOnce":false,"all":false,"but":false,"noaudio":false,"novids":false,"simul":false}
[S.1151] Uta no Prince-sama
  (24631) [E1] Episode 1 - Op.1 La Boussole aux sept couleurs
  (24632) [E2] Episode 2 - Op.2 Brand New Melody
  (24633) [E3] Episode 3 - Op.3 Knocking on the mind
  (24634) [E4] Episode 4 - Op.4 Jusqu'au bout du monde, Believe Heart
  (24635) [E5] Episode 5 - Op.5 Virilité ultime ! Go! Fight!!
  (24636) [E6] Episode 6 - Op. 6 Shout out sur Orion
  (24637) [E7] Episode 7 - Op. 7 Feeling Heart
  (24638) [E8] Episode 8 - Op. 8 Eternity Love
  (24639) [E9] Episode 9 - Op. 9 Nos rêves tissés
  (24640) [E10] Episode 10 - Op. 10 Allez, let's song !
  (24641) [E11] Episode 11 - Op. 11 Le Plan du futur
  (24642) [E12] Episode 12 - Op. 12 Mon coeur égaré
  (24643) [E13] Episode 13 - Op. 13 Love à 1 000 %
  (24644) [E1] Episode 1 - Poison Kiss
  (24645) [E2] Episode 2 - Réincarnation de l'amour
  (24646) [E3] Episode 3 - True Wing
  (24647) [E4] Episode 4 - Les Cerisiers de l'amour
  (24648) [E5] Episode 5 - Smile Magic
  (24649) [E6] Episode 6 - Rhapsodie orange
  (24650) [E7] Episode 7 - Serment à Sirius
  (24651) [E8] Episode 8 - La Fantaisie des étoiles
  (24652) [E9] Episode 9 - We are Starish
  (24653) [E10] Episode 10 - Crystal Time
  (24654) [E11] Episode 11 - Symphonie des rêveurs
  (24655) [E12] Episode 12 - Heavens Gate
  (24656) [E13] Episode 13 - 2 000 % Love
  (24658) [E1] Episode 1 - The dice are cast
  (24659) [E2] Episode 2 - Golden star
  (24660) [E3] Episode 3 - Emotional Life
  (24661) [E4] Episode 4 - Innocent Wind
  (24662) [E5] Episode 5 - Code:T.V.U
  (24663) [E6] Episode 6 - Saintly Territory
  (24664) [E7] Episode 7 - Only One
  (24665) [E8] Episode 8 - Original Resonance
  (24666) [E9] Episode 9 - Never…
  (24667) [E10] Episode 10 - Answer
  (24668) [E11] Episode 11 - Thank you
  (24669) [E12] Episode 12 - Revolution Eve
  (24670) [E13] Episode 13 - Love Revolutions
  (24671) [E1] Episode 1 - Chanter ses rêves
  (24672) [E2] Episode 2 - Liens
  (24673) [E3] Episode 3 - Mighty Aura
  (24674) [E4] Episode 4 - Justice Impulse
  (24675) [E5] Episode 5 - Visible Elf
  (24676) [E6] Episode 6 - Lovely Eyes
  (24677) [E7] Episode 7 - Grown Empathy
  (24678) [E8] Episode 8 - Lasting Oneness
  (24679) [E9] Episode 9 - Next Door
  (24680) [E10] Episode 10 - We believe you
  (24681) [E11] Episode 11 - Merci pour les rêves, le futur et…
  (24682) [E12] Episode 12 - Three Shining Stars
  (24683) [E13] Episode 13 - We are STARISH!

------------------------------

✓ [S1E1] Op.1 La Boussole aux sept couleurs
✓ [S2E1] Poison Kiss
✓ [S3E1] The dice are cast
✓ [S4E1] Chanter ses rêves
[DEBUG] Got download options: {"id":"1151","q":2,"e":"1","dubLang":["jpn"],"dlsubs":["fr"],"fileName":"[${service}] ${showTitle} - S${season}E${episode} [${height}p]","dlVideoOnce":false,"all":false,"but":false,"noaudio":false,"novids":false,"simul":false,"ids":[24631],"title":"Uta no Prince-sama - Episode 1","parent":{"title":"Uta no Prince-sama","season":"1"},"image":"https://image.animationdigitalnetwork.fr/license/utapuri/tv/web/eps1_320x180.jpg","episode":"1"}
[S.1151] Uta no Prince-sama
  (24631) [E1] Episode 1 - Op.1 La Boussole aux sept couleurs
  (24632) [E2] Episode 2 - Op.2 Brand New Melody
  (24633) [E3] Episode 3 - Op.3 Knocking on the mind
  (24634) [E4] Episode 4 - Op.4 Jusqu'au bout du monde, Believe Heart
  (24635) [E5] Episode 5 - Op.5 Virilité ultime ! Go! Fight!!
  (24636) [E6] Episode 6 - Op. 6 Shout out sur Orion
  (24637) [E7] Episode 7 - Op. 7 Feeling Heart
  (24638) [E8] Episode 8 - Op. 8 Eternity Love
  (24639) [E9] Episode 9 - Op. 9 Nos rêves tissés
  (24640) [E10] Episode 10 - Op. 10 Allez, let's song !
  (24641) [E11] Episode 11 - Op. 11 Le Plan du futur
  (24642) [E12] Episode 12 - Op. 12 Mon coeur égaré
  (24643) [E13] Episode 13 - Op. 13 Love à 1 000 %
  (24644) [E1] Episode 1 - Poison Kiss
  (24645) [E2] Episode 2 - Réincarnation de l'amour
  (24646) [E3] Episode 3 - True Wing
  (24647) [E4] Episode 4 - Les Cerisiers de l'amour
  (24648) [E5] Episode 5 - Smile Magic
  (24649) [E6] Episode 6 - Rhapsodie orange
  (24650) [E7] Episode 7 - Serment à Sirius
  (24651) [E8] Episode 8 - La Fantaisie des étoiles
  (24652) [E9] Episode 9 - We are Starish
  (24653) [E10] Episode 10 - Crystal Time
  (24654) [E11] Episode 11 - Symphonie des rêveurs
  (24655) [E12] Episode 12 - Heavens Gate
  (24656) [E13] Episode 13 - 2 000 % Love
  (24658) [E1] Episode 1 - The dice are cast
  (24659) [E2] Episode 2 - Golden star
  (24660) [E3] Episode 3 - Emotional Life
  (24661) [E4] Episode 4 - Innocent Wind
  (24662) [E5] Episode 5 - Code:T.V.U
  (24663) [E6] Episode 6 - Saintly Territory
  (24664) [E7] Episode 7 - Only One
  (24665) [E8] Episode 8 - Original Resonance
  (24666) [E9] Episode 9 - Never…
  (24667) [E10] Episode 10 - Answer
  (24668) [E11] Episode 11 - Thank you
  (24669) [E12] Episode 12 - Revolution Eve
  (24670) [E13] Episode 13 - Love Revolutions
  (24671) [E1] Episode 1 - Chanter ses rêves
  (24672) [E2] Episode 2 - Liens
  (24673) [E3] Episode 3 - Mighty Aura
  (24674) [E4] Episode 4 - Justice Impulse
  (24675) [E5] Episode 5 - Visible Elf
  (24676) [E6] Episode 6 - Lovely Eyes
  (24677) [E7] Episode 7 - Grown Empathy
  (24678) [E8] Episode 8 - Lasting Oneness
  (24679) [E9] Episode 9 - Next Door
  (24680) [E10] Episode 10 - We believe you
  (24681) [E11] Episode 11 - Merci pour les rêves, le futur et…
  (24682) [E12] Episode 12 - Three Shining Stars
  (24683) [E13] Episode 13 - We are STARISH!

------------------------------

✓ [S1E1] Op.1 La Boussole aux sept couleurs
✓ [S2E1] Poison Kiss
✓ [S3E1] The dice are cast
✓ [S4E1] Chanter ses rêves
[ERROR] 500: Internal Server Error
[ERROR] {"message": "Internal server error"}
[ERROR] Token refresh failed!
[ERROR] Failed to refresh token
[ERROR] Failed to download media list
[ERROR] 500: Internal Server Error
[ERROR] {"message": "Internal server error"}
[ERROR] Token refresh failed!
[ERROR] Failed to refresh token
[ERROR] Failed to download media list
[ERROR] 500: Internal Server Error
[ERROR] {"message": "Internal server error"}
[ERROR] Token refresh failed!
[ERROR] Failed to refresh token
[ERROR] Failed to download media list
[ERROR] 500: Internal Server Error
[ERROR] {"message": "Internal server error"}
[ERROR] Token refresh failed!
[ERROR] Failed to refresh token
[ERROR] Failed to download media list
```

**Comment by Jaynator495** [comment]
Should be fixed in the latest commit, just waiting for someone to test it and confirm

**Comment by nigoki2** [comment]
Thank you. I'm waiting fot the new windows gui.

**Comment by AnimeDL** [comment]
Finally released version 5.1.5 which should fix this issue. Sorry for taking so long to get this out
